packages/cli/internal/pkg/cli/workflow/workflow_run.go

```diff
@@ -11,17 +11,19 @@ func (m *Manager) RunWorkflow(contextName, workflowName, inputsFileUrl string, o
 	m.validateContextIsDeployed(contextName)
 	m.setOutputBucket()
 	m.parseWorkflowLocation()
+	m.readInput(inputsFileUrl)
+	m.initializeTempDir()
+	m.writeTempManifest()
+	m.uploadInputsToS3()
+	m.parseInputToArguments()
 	if m.isUploadRequired() {
```

Comment:
qq: is `isUploadRequired` logic needed to be updated to not only upload when there is a MANIFEST.JSON but also a `--inputFiles`:

Reply:
so if there is no MANIFEST.json, should a new one be created in that case?

Reply:
Yes. considering that all required inputFiles are centralized in MANIFEST.JSON, we should also update a copy if there is only inputFiles. Therefore, you should double check the logic.